tests/syft/lib/petlib/ec_obj_test.py

```diff
@@ -0,0 +1,35 @@
+# Testing code
+
+# third party
+import petlib
```

Comment:
this is breaking CI because petlib doesnt exist until the vendor lib install step, so you need to move it inside the test and make sure the test has a @pytest.vendor(lib="petlib") tag around it.